airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -496,15 +496,12 @@ def __init__(
         self.tags = tags
         self.awslogs_group = awslogs_group
         self.awslogs_stream_prefix = awslogs_stream_prefix
-        self.awslogs_region = awslogs_region
+        self.awslogs_region = awslogs_region or self.region
         self.awslogs_fetch_interval = awslogs_fetch_interval
         self.propagate_tags = propagate_tags
         self.reattach = reattach
         self.number_logs_exception = number_logs_exception
 
-        if self.awslogs_region is None:
-            self.awslogs_region = self.region
```

Comment:
I wonder why the condition was `is None`, should empty string be accepted?

Reply:
Empty region_name is invalid region

```
import boto3

session = boto3.session.Session(region_name="")
session.client("sts").get_caller_id()

# ValueError: Invalid endpoint: https://sts..amazonaws.com
```